scripts/src/java/org/oppia/android/scripts/common/model/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+"""
+Data structures corresponding to common utilities, such as for GitHub API integration.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
+
+kt_jvm_library(
+    name = "github_issue",
+    testonly = True,
```

Comment:
This actually has nothing to do with production vs. tests. It was a hack I introduced back when the first scripts were added to ensure that production _app_ code can't depend on scripts dependencies (since they run in a JVM environment and are allowed completely different transitive dependencies from Android). It's confusing, and semantically wrong.

It will also be fixed in #4920 (which is a ways away from being updated & ready for review).